astropy/constants/si.py

```diff
@@ -6,157 +6,329 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+import inspect
 import numpy as np
 
 from .constant import Constant, EMConstant
 
+
+def get_attributes(a_class):
+    attributes = inspect.getmembers(a_class,
+                               lambda a:not(inspect.isroutine(a)))
+    return [a for a in attributes
+                if not(a[0].startswith('__') and a[0].endswith('__'))]
+
 # PHYSICAL CONSTANTS
 
-# Planck constant
-h = Constant('h', "Planck constant", 6.62606957e-34, 'J s', 0.00000029e-34,
-             'CODATA 2010', system='si')
+class CODATA2014:
```

Comment:
I am not comfortable with using old-style class syntax for Python 2, while we still support Python 2.